packages/plugins/response-cache/test/response-cache.spec.ts

```diff
@@ -1482,7 +1482,7 @@ describe('useResponseCache', () => {
     expect(spy).toHaveBeenCalledTimes(1);
   });
 
-  it('cache is purged upon mutation even when error is included in the mutation execution result', async () => {
+  it.skip('cache is purged upon mutation even when error is included in the mutation execution result', async () => {
```

Comment:
Note that we can no longer do this as we only operate on the complete execution result compared to an resolver level before.